### PR TITLE
Getting started with Node.js 15

### LINKED ISSUE #1999: Introduction to Node.js 15
### Proposed title of article
Introduction to Node.js 15

### Introduction paragraph (2-3 paragraphs):
Node.js is a popular framework for executing JavaScript code on the server. This functionality allows developers to build powerful and scalable applications. The release of Node.js 15 takes things to a new level . 

Production deployments are generally recommended to use LTS release line.
Node.js 15 has distinctive features such as abort controller, N-API version 7 and many others. One of the major changes in  Node.js 15 is that peer dependencies are installed by default.

### Key takeaways: 
-Understand what node.js 15 entails 
-Understand the packages required in node.js 15
-understand the key advantages of node.js 15 over other node.js versions
### References:
 N|A
### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by valerio140**
### Article checklist
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [ ] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

This fixes #1999

### CODE REVIEW

**Comment by paulodhiambo** [comment]
@valerio140 I can't an issue for this PR. please comment with the issue number so that I can link it to this PR.

**Comment by valerio140** [comment]
> @valerio140 I can't an issue for this PR. please comment with the issue number so that I can link it to this PR.

sorry, the issue number is #1999

File: articles/introduction-to-node.js-15/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+### Introduction
+
+So, you've heard of Node.js but don't know what it is or how it works in your development process. Maybe you've heard people praising Node.js and you are now wondering if you should learn it as well. Maybe you're already acquainted with another back-end technology and want to learn more about Node.js. You've come to the right spot.
```
Comment (paulodhiambo) [inline articles/introduction-to-node.js-15/index.md:3]: Consider removing this part, it's unnecessary.

File: articles/introduction-to-node.js-15/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+### Goal
+This article discusses new features in Node.js 15. It will also show why you should consider updating to the latest release.
+
+### New features of Node.js 15
+Node.js 15 is the most recent version, which was released in October of 2020. It doesn’t have long-term support. Output improvements, a V8 JavaScript engine upgrade, bug fixes, and an experimental version of Abort Controller and QUIC are all included.
+
+ **NPM 7**
+
+Thanks to a slew of new features, NPM 7 is likely to be the biggest highlight of this update. NPM workspaces are the most important aspect since they mark the beginning of built-in support for building and managing multiple NPM packages in a single file system. Developers who are already familiar with the Yarn package manager or Lerna should find NPM's workspace implementation to be identical. NPM workspaces are relatively simple to set up. Start by arranging NPM package files on the file system, build a top-level package. Son that refers to each of these files, start running commands through all NPM packages.
```
Comment (paulodhiambo) [inline articles/introduction-to-node.js-15/index.md:15]: Consider changing "... Lerna should find NPM's workspace implementation to be identical." to "... Lerna should find NPM's workspace implementation identical."

**Review by paulodhiambo — CHANGES_REQUESTED** [review]
@valerio140 amazing article. Review the changes and ping me when done. 
**Quick note** 
Make sure to be using active voice instead of passive voice when writing articles.

**Comment by valerio140** [comment]
@paulodhiambo I am done with the revision. Thanks

**Comment by paulodhiambo** [comment]
@valerio140 i can see two index.md files. Why are they two? Please correct.

**Comment by valerio140** [comment]
> @valerio140 I can see two index.md files. Why are they two? Please correct.

@paulodhiambo Please use Update index.md. It was  mistake  while uploading. Thanks

**Comment by paulodhiambo** [comment]
@valerio140 delete content/articles/why-you-should-use-wakanda-javascript-framework/index.md

**Comment by valerio140** [comment]
@paulodhiambo file deleted

**Comment by hectorkambow** [comment]
plagiarism check done

File: articles/introduction-to-node.js-15/index.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+- V8 JavaScript engine upgrade.
+- Bugfix and an experimental version of Abort Controller and QUIC are all included.
+
+ **NPM 7**
+
+Thanks to a slew of new features, NPM 7 is likely to be the biggest highlight of this update. NPM workspaces are the most important aspect since they mark the beginning of built-in support for building and managing multiple NPM packages in a single file system. Developers who are already familiar with the Yarn package manager or Lerna should find NPM's workspace implementation identical. NPM workspaces are relatively simple to set up. Start by arranging NPM package files on the file system, build a top-level package. Son that refers to each of these files, start running commands through all NPM packages.
+
+**V8 8.6**
+
+Node.js 15's V8 engine has been updated to version 8.6 from version 8.4. V8 is the JavaScript engine that runs Node.js. Since V8, the features of the JavaScript language that are available to programmers have been specified. Despite the small version bump, there are a few new language enhancements that will help you write less code and potentially increasing the readability of your code.
```
Comment (paulodhiambo) [inline articles/introduction-to-node.js-15/index.md:41]: Explain this section. Its not clear.

**Review by paulodhiambo — CHANGES_REQUESTED** [review]
@valerio140 Revise the requested changes and ping me when done.

**Comment by paulodhiambo** [comment]
@valerio140 amazing article. @hectorkambow you can give it a final review

**Comment by paulodhiambo** [comment]
Hello :wave:
As you know, we were working on a new structure for our EngEd Repo, which has led to a slight slow down of article publishing. Everything is almost ready now.
In order to publish your article as soon as possible, please do the following:
- Download the most recent files off this PR (Make sure to pick the latest one with my revisions and edits).
- Delete your article directory from the *articles* folders.
- Do a git pull (take a look at the new structure of the repo after a git pull).
- Upload your article files to the [new *articles* folder inside *content*](https://github.com/section-engineering-education/engineering-education/tree/master/content/articles).

Ping me via Slack if you have any problems.

**Comment by hectorkambow** [comment]
Thank you v much for your patience as we worked through our Repo re-structuring process. 

The purpose behind this restructure - was to better allow us to handle more volume and allow students to publish more articles and get more involved with the community (with open-source contribution and suggestions). 

With most of the changes coming to completion - we will be sure to get your article published asap. 

Thank you again - and we are looking forward to more amazing articles.

**Pull request closed without merging**